src/olympia/amo/tests/test_storage_utils.py

```diff
@@ -4,7 +4,6 @@
 
 from django.conf import settings
 from django.core.files.base import ContentFile
-from django.utils.encoding import force_str
```

Comment:
I'm guessing this method is what was causing us the problems from the upgrade. It would be worth it IMO to add a quick blurb linking a snippet of this, what you replaced with and any link to a note in the release docs (again, if it exists) but at least so we can remember in the future why we cannot use this method.

Reply:
I don't believe that was the cause (it seemed obsolete) - afaik it was because we were passing absolute paths to the save function and they added another check for the filepath that baulks on absolute paths (because potential for directory traversal) _before_ the function that would have normalized the absolute path into a relative one to the base folder of the Storage.

Reply:
i.e. https://github.com/django/django/commit/2b00edc0151a660d1eb86da4059904a0fc4e095e#diff-2f8524035203a28c596eb210d5b886a12d5a46215a649bee85f9987cc335d128R41 - `validate_filename` doesn't like absolute paths.